### ISSUE TITLE
Request to add support for Chinese

### ISSUE THREAD

**Issue by tyf2018** [tadashi-aikawa/obsidian-various-complements-plugin#7]
Great plugin. Have you considered adding support for Chinese? Thank you!

**Comment by tadashi-aikawa** [comment]
Hi, @tyf2018. Thanks for your feature request!

I'll try to implement it by using a [chinese-tokenizer](https://github.com/yishn/chinese-tokenizer). However, since I can't read Chinese, it would be great to check if it is correct when I implemented it. 😄

**Comment by lbgws2** [comment]
I am also Chinese. It will be convenient to use my mother tongue in the plug-in

**Comment by tadashi-aikawa** [comment]
I'll continue to try to implement it in v7.0.0 👍 

https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/discussions/133

**Comment by tadashi-aikawa** [comment]
```
console.log(tokenize("我是中国人。"));
console.log(tokenize("我认为 Obsidian 是编写文档的好工具。"));
```

I cannot decide if the below is a good result or not. But looks good? 🤔 

![image](https://user-images.githubusercontent.com/9500018/170823782-0d438f94-dec1-47f7-90cc-a456b4e74fb7.png)

**Comment by tadashi-aikawa** [comment]
Hi, @tyf2018 @lbgws2 

I have implemented **Chinese** strategy experimental :)
 
https://user-images.githubusercontent.com/9500018/170825005-10738fd5-c67e-4991-83f7-1eda391d0392.mp4

Note: I used Japanese IME, so it didn't input smoothly 😅 

If you are OK, I will release it as `v7.0.0-beta1`. What do you think?

**Comment by tyf2018** [comment]
> If you are OK, I will release it as `v7.0.0-beta1`. What do you think?

Wow, that's great, looks like the support for Chinese is better!

**Comment by tadashi-aikawa** [comment]
@tyf2018 
Released [v7.0.0-beta1](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/discussions/133#discussion-4104744) 🚀  (need BRAT)

It would be helpful to give me some feedback 😄

**Comment by lbgws2** [comment]
I have tried 7.0.0-beta1, which is pretty-good,

Great work, thank you!

**Comment by tyf2018** [comment]
I have tried 7.0.0-beta1 to beta7, currently using beta7, and now this plugin has great Chinese support!

**Comment by tadashi-aikawa** [comment]
Released in [v7.0.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/7.0.0) 🚀
